docs/architecture/dapr-for-net-developers/service-invocation.md

```diff
@@ -105,7 +105,7 @@ Alternatively, you can configure a custom endpoint in the call to `DaprClient.Cr
 var httpClient = DaprClient.CreateHttpClient(daprEndpoint = "localhost:4000");
```

Comment:
```suggestion
var httpClient = DaprClient.CreateInvokeHttpClient(daprEndpoint = "localhost:4000");
```

See, https://github.com/dotnet/docs/pull/24652 - I'm curious if this comes from an extension method or what? But we should figure that out and address this.

Reply:
Not an extension method, just a typo unfortunately :-(
Search through the entire text, #24652 fixed them all.